src/terminal_view/DecorationRenderer.cpp

```diff
@@ -232,7 +233,23 @@ void DecorationRenderer::rebuild()
             move(image)
         );
     } // }}}
-    // TODO: CrossedOut
+    { // {{{ crossed-out
```

Comment:
I think this came in by accident.

Reply:
I was adding the framed feature while working on a branch from the old master (before crossed-out was added).
So I'm trying to get my master updated with the crossed out feature and then combine that with the framed feature.

Reply:
I'm having trouble squashing these 2 commits into 1 since it contains a merge. Perhaps this wasn't the best way of doing things?

Reply:
Do you happen to remember how you did end up in this state? Just so I understand. Did you do something like: `git merge master` while being in your working branch? I think the right command should have been `git fetch && git rebase origin/master`, ideally.

I think I can provide you a small video (or session tomorrow morning on Twitch) on how to unfuck the situation.

Reply:
Yes I did a git merge master while in the working branch. Then I had merge conflict on the DecorationRenderer.cpp file. So I opened it up and just deleted the lines that looked like `<<<<<<< HEAD`, `=======`, and `>>>>>>>` so the file looked correct. Then applied your diff `git apply patch.diff` and pushed.